templates/al2023/provisioners/install-neuron-driver.sh

```diff
@@ -29,4 +29,4 @@ metadata_expire=0" |sudo tee /etc/yum.repos.d/neuron.repo
 ################################################################################
 ### Install packages ###########################################################
 ################################################################################
-sudo dnf --setopt=install_weak_deps=False install -y aws-neuronx-dkms
\ No newline at end of file
+sudo dnf install -y aws-neuronx-dkms
```

Comment:
I think adding the `aws-neuronx-tools ` would be a worthy 140mb - the Neuron AMI should be pretty lightweight as is. You can think of it as `nvidia-smi` for Neuron somewhat

Reply:
sounds good, will add it